projects/ghostscript/gstoraster_fuzzer.cc

```diff
@@ -1,6 +1,20 @@
-// Copyright 2019 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+/*
+# Copyright 2019 The Chromium OS Authors.
```

Comment:
nit: remove the # from all the lines

Reply:
Can do, but this is exactly how it looks in [MuPDF](https://github.com/google/oss-fuzz/blob/master/projects/mupdf/pdf_fuzzer.cc). Do you want me to remove the hasmarks from both?